runner/run_benchmark.py

```diff
@@ -121,14 +121,16 @@ def generate_test_examples(
 
 def check_result_files(run_name: str, agent_name: str, force_removal: bool = False, pass_default: bool = False):
     result_files = gather_result_files(run_name, agent_name)
+    persistence_files = gather_persistence_files(run_name, agent_name)
+    all_files = result_files + persistence_files
     if force_removal:
-        for file in result_files:
+        for file in all_files:
             os.remove(file)
-        result_files = []
-    if len(result_files) > 0:
+        all_files = []
+    if len(all_files) > 0:
         if not pass_default:
             if not ask_yesno(
-                f"{len(result_files)} result files have been found for run name {run_name} " f"and agent {agent_name}.",
+                f"{len(all_files)} result files have been found for run name {run_name} " f"and agent {agent_name}.",
```

Comment:
This is fine from a functional standpoint, but I think the wording should change. Something like
```python
f"There is data from a past run with {len(result_files)} results collected."
```